frontend/server/src/DAO/Tags.php

```diff
@@ -75,4 +75,46 @@ public static function findPublicTagsByPrefix(
 
         return $results;
     }
+
+    /**
+     * @return list<array{name: string}>
+     */
+    public static function getFrequentsByLevel(
+        string $problemLevel
+    ) {
+        $sql = '
+            SELECT
+                t.name
```

Comment:
Este cambio ayudaría a dejar libre el parámetro `name` para la traducción del tag que se almacena en la DB
```suggestion
                t.name AS alias
```